.travis.yml

```diff
@@ -23,3 +23,7 @@ install:
 script:
   - ./pants --changed-since=origin/master lint
   - ./pants --changed-since=origin/master --changed-include-dependees=transitive test
+  # Smoke test that our release process will work.
+  - ./pants binary helloworld/main.py helloworld/main_py2.py
```

Comment:
We should also actually run these binaries (once they run successfully again).

Reply:
Yes, agreed. This is a TODO in the PR description.